app/support/auth/src/main/kotlin/com/xorker/xpr/support/auth/security/TokenAuthenticationFilter.kt

```diff
@@ -30,7 +30,7 @@ internal class TokenAuthenticationFilter(
 
     private fun getAccessToken(request: HttpServletRequest): String? {
         val accessToken = request.getHeader(HEADER_AUTHORIZATION)
-        if (accessToken.isNullOrBlank() && accessToken.startsWith(HEADER_BEARER)) {
+        if (!accessToken.isNullOrBlank() && accessToken.startsWith(HEADER_BEARER)) {
```

Comment:
p5. ! 가시성이 좋지 않아서 개인적으로 .not() 을 선호해서 코멘트 적어봅니다.

```suggestion
        if (accessToken.isNullOrBlank().not() && accessToken.startsWith(HEADER_BEARER)) {
```